lib/libutee/tee_api_operations.c

```diff
@@ -1418,11 +1420,6 @@ TEE_Result TEE_AEEncryptFinal(TEE_OperationHandle operation,
 		goto out;
 	}
 
-	if (!srcData && !srcLen) {
-		res = TEE_SUCCESS;
-		goto out;
-	}
-
```

Comment:
Yeah, of course. Is `memcpy(p, NULL, 0)` valid? I think it is, but Coverity must not know about it I guess. So, change `tee_buffer_update()` maybe?